ddtrace/span.py

```diff
@@ -110,6 +113,7 @@ def __init__(
         context=None,  # type: Optional[Context]
         on_finish=None,  # type: Optional[List[Callable[[Span], None]]]
         span_api=SPAN_API_DATADOG,  # type: str
+        links=None,  # type: Optional[List[_span_link.SpanLink]]
```

Comment:
```suggestion
        _links=None,  # type: Optional[List[_span_link.SpanLink]]
```

? does it need to be public? (I have no idea, just commenting to be protective)

Reply:
I could make this private since users are not expected to initialize a span object directly. This argument will be mainly used internally (ex: in `tracer.trace(..., links=[SpanLink(..), SpanLink(...), ....])`). 

No strong feelings here